docs/banzai/index.rst

```diff
@@ -151,59 +187,116 @@ has dimensions that are half of the full image).
+the absolute values along the vertical axis. Next, the SNR is computed as:
 
-Currently no non-linear distortion is included in the WCS (the current WCS solution only has a center,
-a pixel scale, and a rotation). At worst (in the image corners), the offset between
-coordinates with non-linear distortion terms included and those without are ~5 arcseconds.
+SNR = [power - median(power)] / MAD(power)
 
-Eventually, astrometry.net may be replaced with a purely Python alternative
-(e.g. AliPy; http://obswww.unige.ch/~tewes/alipy/).
+The frame is considered to have pattern noise if more than 5 pixels are above an SNR of 15.
+
+Pointing Test
+=============
+This test computes the offset between the requested RA and declination from the header
+(given by either ``OFST-RA`` and ``OFST-DEC``, or ``CAT-RA`` and ``CAT-DEC``)
+with the actual RA and declination of the observation (``CRVAL1`` and ``CRVAL2``).
+The test is considered failed if the offset is above 300", and a warning is provided if it is above 30".
+
+
+Master Calibration Comparison
+=============================
+
+When a calibration frame is processed by BANZAI, it can be compared to the temporally nearest
+previous master to check
+for significant variations, which can serve as an alert for e.g. major issues with the camera.
+Since this check also discards frames found to deviate significantly,
+it prevents the creation of bad master frames that can cause
+problems as they are propagated through the pipeline and used for the reduction of science data.
+
+The algorithm works as follows.  After some preprocessing that depends on the calibration type,
+the SNR at each pixel is computed as:
+
+SNR = (individual_frame - master_frame) / noise
+
+where the noise also depends on the type of calibration.
+The individual frame fails the comparison test if more than 5% of pixels have an SNR greater than 6.
+
+The individual frame preprocessing steps and noise parameters for the different calibration types are listed below:
+
+- bias:
+
+  - preprocessing: bias level subtraction
+  - noise = RN (read noise, from header keyword ``RDNOISE``)
+
+- dark:
+
+  - preprocessing: bias subtraction, normalization by exposure time
+  - noise = sqrt( RN\^2 + PN\^2) / exptime, where PN is the poisson noise, computed using the
+    square root of the image counts prior to normalization
+
+- skyflat:
+
+  - preprocessing: bias and dark subtraction, normalization by the sigma clipped mean of image
+  - noise = sqrt( RN\^2 + PN\^2) / normalization
+
+
+Other Considerations
+--------------------
+
+
+Sinistro Cameras
+================
+While the Spectral and SBIG cameras produce single extension fits files, the Sinistro frames produce
+four amplifier outputs.
+The top amplifiers (3 and 4) have more light sensitive pixels than the bottom amplifiers (1 and 2). All of the parallel
+overscan rows are discarded. One edge row is discarded for the amplifiers that have an extra light sensitive row.
```

Comment:
This requires more context for outside users. I would just say that in historic data, there was a missing row in the center because the top amplifiers... The Sinistro frames are now correctly mosaiced. We already say above we trim the images so the rest is not really necessary.

Reply:
Changed